buggy_utils.py

```
@@ -0,0 +1,107 @@
+    query = "SELECT * FROM users WHERE name = '" + username + "'"
+    return conn.execute(query).fetchall()
+
+
+# Command injection vulnerability
+def run_diagnostic(host):
+    result = os.system("ping -c 1 " + host)
+    return result
+
+
+# Hardcoded credentials
+DB_PASSWORD = "admin123"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "DB_PASSWORD"

(S105)